hack/build-cross.sh

```diff
@@ -20,6 +25,8 @@ OS_BUILD_PLATFORMS=("${platforms[@]}")
 host_platform=$(os::build::host_platform)
 if [[ $host_platform == "linux/ppc64le" ]]; then
   OS_GOFLAGS_LINUX_PPC64LE="-tags=gssapi" os::build::build_binaries "${OS_CROSS_COMPILE_TARGETS[@]}"
+elif [[ $host_platform == "linux/arm64" ]]; then
```

Comment:
Sorry for the delay, but my aarch64 testing area is having issues. 
On x86_64 this works if you get rid of the 'local platform_goflags_envvar;'  So it looks like this.

```
platform_goflags_envvar="OS_GOFLAGS_$(echo ${platform} | tr '[:lower:]/' '[:upper:]_')"
declare "${platform_goflags_envvar}=-tags=gssapi"
os::build::build_binaries "${OS_CROSS_COMPILE_TARGETS[@]}"
```
If it passes the aarch64 test, I'll update this pull request with it in.